src/App.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import React, { useState } from 'react';
+import cn from 'classnames';
```

Comment:
For the future, better not to use such short names, you can name it `classnames` here